cocos/spine/assembler/simple.ts

```diff
@@ -36,6 +36,7 @@ import { Color, EPSILON, Vec3 } from '../../core';
 import type { MaterialInstance } from '../../render-scene';
 import type { IBatcher } from '../../2d/renderer/i-batcher';
 
+const ADJUST_SIZE_RATE = 1.1;
```

Comment:
If Spine instance counts exceeding thresholds cause issues, first adjust the ADJUST_SIZE_RATE value.
Going forward, we need to implement independent memory management for Spine that bypasses the chunk approach - similar to native IOBuffers where populated content transfers directly to GPU buffers without going through static_vb_accessor. For now, provide the minimal viable change.